src/main/java/create_task_use_case/CreateTaskInteractor.java

```diff
@@ -3,39 +3,43 @@
  * Author: Fardin Faruk
  * Modified By: Fardin Faruk
  * Created: Nov 19, 2022
- * Last Modified: Nov 26, 2022
+ * Last Modified: Nov 27, 2022
  */
 package create_task_use_case;
 
 import entities.Task;
 import entities.TaskFactory;
 
+import java.lang.reflect.Array;
+
 public class CreateTaskInteractor implements CreateTaskInputBoundary {
-    final CreateTaskDsGateway createTaskDsGateway;
-    final CreateTaskOutputBoundary createTaskOutputBoundary;
+    final CreateTaskDsGateway dsGateway;
+    final CreateTaskOutputBoundary outputBoundary;
     final TaskFactory taskFactory;
 
-    public CreateTaskInteractor(TaskFactory taskFactory, CreateTaskOutputBoundary createTaskOutputBoundary,
-                                CreateTaskDsGateway createTaskDsGateway){
+    final private String[] DAYSOFWEEK = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
```

Comment:
Good solution to the issue of relating day names to indexes for the output data. I'll implement this in my use case as well.

Reply:
Sounds good